### ISSUE TITLE
not run

### ISSUE THREAD

**Issue by mianfazal** [karkranikhil/face-recognition-using-js#1]
give error during run it on local computer....plz explain me how it run..
it show just html body and tags,,
even not capturing webcam
![Capture](https://user-images.githubusercontent.com/61380504/77225563-6b880c80-6b2d-11ea-810d-46914647f3b5.PNG)

**Comment by MartinBspheroid** [comment]
@mianfazal -> You should probably explain to @karkranikhil what is going on behind the scenes, e.g. using the Developer tools.

**Comment by MatheusLealAquino** [comment]
I had same issue, but is because the project is not running in "localhost". Is necessary initialize a server to run the html file. (Explanation: https://blog.mozilla.org/webrtc/camera-microphone-require-https-in-firefox-68/)
I ran with python on Ubuntu with the command "python -m SimpleHTTPServer" inside the folder where the index.html file is located

**Comment by MartinBspheroid** [comment]
Yeah and also remember to do [monkeyPatch](https://github.com/justadudewhohacks/face-api.js/issues/157#issuecomment-443507577) in order to run in Electron environment.

**Comment by thienson30** [comment]
I have problem ... Webcam doesn't load at all... Your example is work fine on my browser firefox and chrome; however, I git clone yours and put on my server at https://elpasonails.com/face-recognition-using-js/index.html and it doesn't work.  Can you take a look at it and let me know what wrong?

Thanks in advance

**Comment by thienson30** [comment]
> give error during run it on local computer....plz explain me how it run..
> it show just html body and tags,,
> even not capturing webcam
> ![Capture](https://user-images.githubusercontent.com/61380504/77225563-6b880c80-6b2d-11ea-810d-46914647f3b5.PNG)

you have to run on secure server... local server in general will not work
